Dockerfile

```diff
@@ -11,10 +11,12 @@ ENV LANG C.UTF-8
 ENV PYTHON_VERSION_27 2.7.14
 ENV PYTHON_VERSION_35 3.5.5
 ENV PYTHON_VERSION_36 3.6.4
-ENV CONDA_VERSION 4.4.10
+ENV PYTHON_VERSION_37 3.7.0
```

Comment:
Note from [Python 3.7 release schedule](https://www.python.org/dev/peps/pep-0537/):

> 3.7.1 final: 2018-10-18 (expected)

We might want to hold this one for a bit?

Reply:
I suppose, yes.

Reply:
Ok! I'll update this PR when 3.7.1 gets released